js/activesync/account.js

```diff
@@ -368,7 +370,7 @@ ActiveSyncAccount.prototype = {
   },
 
   shutdown: function asa_shutdown(callback) {
-    this._LOG.__die();
+    logic(this, 'die');
```

Comment:
We should probably plan on removing these or changing the string to "shutdown" or something like that.  The previous log-lifecycle stuff did not print out anything when the logger's life-cycle ended.  Now the email app might keep spitting out "die" a lot which is at once potentially humorous and potentially confusing/concerning.  (Especially to those who think the email app has a problem with them!)

Because of the potential confusion, this needs at least a spin-off bug if not addressed in this pull request.